include/nanogui/glutil.h

```diff
@@ -520,18 +736,57 @@ struct Arcball {
         return true;
     }
 
+    /**
+     * Returns the current rotation *including* the active motion, suitable for
+     * use with typical homogeneous matrix transformations.  The upper left 3x3
+     * block is the rotation matrix, with 0-0-0-1 as the right-most column /
+     * bottom row.
+     */
     Matrix4f matrix() const {
         Matrix4f result2 = Matrix4f::Identity();
         result2.block<3,3>(0, 0) = (mIncr * mQuat).toRotationMatrix();
         return result2;
     }
 
+    /**
+     * \brief Interrupts the current Arcball motion by calling
+     *        \ref Arcball::button with \ref mLastPos and ``false``.
+     *
+     * Use this method to "close" the state of the Arcball when a mouse release
+     * event is not available.  You would use this method if you need to stop
+     * the Arcball from updating its internal rotation, but the event stopping
+     * the rotation does **not** come from a mouse release.  For example, you
+     * have a callback that created a \ref nanogui::MessageDialog which will now
+     * be in focus.
+     */
+    void interrupt() { button(mLastPos, false); }
```

Comment:
Insofar as I understand, it truly doesn't matter if we use `button(mLastPos, false)` since it is a "release".  However, would it be more correct to do `button(Vector2i::Zero(), false)` instead?